test/mocha/050-client-api.js

```diff
@@ -3,20 +3,42 @@
  */
 'use strict';
 
-const {_client} = require('bedrock-ledger-consensus-continuity');
+const {_client, _localPeers} = require('bedrock-ledger-consensus-continuity');
 const {config, util: {BedrockError}} = require('bedrock');
 
 describe('Client API', () => {
   describe('notifyPeer', () => {
+    // NOTE: this name is really bad. peerId is actually a database record
+    // with tons of information including private key material
+    let peerId = null;
+    const ledgerNodeId = 'foo';
+    before(async () => {
+      ({peerId} = await _localPeers.generate({ledgerNodeId}));
+    });
+    it('throws a NotFoundError if peerId is not found', async () => {
```

Comment:
This is the case where `ledgerNodeId` is not found, right?

Reply:
yes and addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/0c2ae776d0a3bce31ea8530262f5aec4fd5c79e2